interfaces/c/fortran_order_interface_libstr.c

```diff
@@ -341,7 +341,7 @@ int fortran_order_d_ip_ocp_hard_tv(
 		blasfeo_pack_tran_dmat(nx[ii+1], nu[ii], B[ii], nx[ii+1], &hsBAbt[ii], 0, 0);
 		blasfeo_pack_tran_dmat(nx[ii+1], nx[ii], A[ii], nx[ii+1], &hsBAbt[ii], nu[ii], 0);
 		blasfeo_pack_tran_dmat(nx[ii+1], 1, b[ii], nx[ii+1], &hsBAbt[ii], nu[ii]+nx[ii], 0);
-		blasfeo_pack_dvec(nx[ii+1], b[ii], &hsb[ii], 0);
+		blasfeo_pack_dvec(nx[ii+1], b[ii], 0, &hsb[ii], 0);
```

Comment:
In the `blasfeo_pack_dvec`, the vector increment `xi` should be set to 1 when you want to write in consecutive vector elements (and not 0).
With 0, it would keep overwriting the first vector element.

Reply:
Oops, thanks for checking! I just updated the commit with `xi` = `1`.